components/org.wso2.carbon.identity.remotefetch.common/src/main/java/org/wso2/carbon/identity/remotefetch/common/ConfigurationFileStream.java

```diff
@@ -22,14 +22,15 @@
 import java.io.InputStream;
 
 /**
- * Holds raw config file content.
+ * Holds raw config file content for reading xml files to deploy configurations.
+ *
  */
 public class ConfigurationFileStream {
 
     private InputStream contentStream;
     private File path;
 
-    /**
+    /**Default constructor.
```

Comment:
Formatting issue. /** line should be kept without any other text. Please fix all the instances.

Reply:
Fixed